src/vcpkg/binarycaching.cpp

```diff
@@ -1799,7 +1799,7 @@ namespace vcpkg
         if (auto p = args.vcpkg_nuget_repository.get())
         {
             get_global_metrics_collector().track_define(DefineMetric::VcpkgNugetRepository);
-            return {std::move(*p)};
+            return {*p};
```

Comment:
Can you explain this one?

Reply:
`args` is the argument of this function which is passed by const&. Therefore, it's not possible to move the contents of `vcpkg_nuget_repository` out of it.

Reply:
Thanks!